README.md

```diff
@@ -147,26 +147,28 @@ $response = $kevinClient->payment()->initPayment($attr);
 
 ### 2.2 Initiate card payment
 
-```
+```php
 $attr = [
     'Redirect-URL' => 'https://redirect.kevin.eu/payment.html',
     'description' => 'Test',
     'currencyCode' => 'EUR',
     'amount' => '0.01',
-    'cardPaymentMethod' => [
-        'cvc' => '123',
-        'expMonth' => '01',
-        'expYear' => '2036',
-        'number' => '5555555555555555',
-        'holderName' => 'John Titor',
+    'bankPaymentMethod' => [
+        'endToEndId' => '1',
+        'creditorName' => 'John Smith',
+        'creditorAccount' => [
+            'iban' => 'LT144010051005081586'
+        ],
     ],
+    'cardPaymentMethod' => [],
+    'paymentMethodPreferred' => 'card'
```

Comment:
Not a big deal, but I see that we add a trailing comma in our examples. Perhaps it is good to stick with it as well